umbrella/example/cms/cmssw_redhat6.json

```diff
@@ -0,0 +1,27 @@
+{
+	"hardware": {
+		"arch": "x86_64",
+		"cores": "2",
+		"memory": "2GB",
+		"disk": "3GB"
+	},
+	"kernel" : {
+		"name": "linux",
+		"version": ">=2.6.32"
+	},
+	"os": {
+		"name": "Redhat",
+		"version": "6.5",
+		"id": "669ab5ef94af84d273f8f92a86b7907a"
+	},
+	"software": {
+		"cmssw-5.3.11-slc5_amd64": {
+			"mountpoint": "/cvmfs/cms.cern.ch"
+		}
+	},
+	"data": {
+	},
+	"environ": {
+		"FAKEPWD": "/tmp/test"
```

Comment:
Ah ok, I think this example is going to be confusing to new users, since it gives the impression that you are deliberately trying to do something with the current working directory.  If the CMS example doesn't need any particular environment variables, then just leave them out of this example.

Of course, you should give examples of setting environment variables in your manual.